src/viadot/utils.py

```diff
@@ -962,3 +962,47 @@ def anonymize_df(
 
     df.drop(columns=["temp_date_col"], inplace=True, errors="ignore")
     return df
+
+
+def df_converts_bytes_to_int(df: pd.DataFrame) -> pd.DataFrame:
+    """Task to convert bytes values to int.
+
+    Args:
+        df (pd.DataFrame): Data Frame to convert
+
+    Returns:
+        pd.DataFrame: Data Frame after convert
+    """
+    return df.applymap(lambda x: int(x) if isinstance(x, bytes) else x)
```

Comment:
Sorry, just noticed that pandas deprecated `applymap` in 2.x, let's use the new way. Also, is this returning a copy or modifying input df? If modifying - I would suggest to always return a copy instead.

```suggestion
    return df.map(lambda x: int(x) if isinstance(x, bytes) else x)
```

Reply:
Done